simplefs/simplefs.go

```diff
@@ -922,6 +922,10 @@ func (k *SimpleFS) doCopyFromSource(
 	}
 	defer dst.Close()
 
+	if pathType, _ := destPath.PathType(); pathType == keybase1.PathType_LOCAL {
+		defer Quarantine(ctx, destPath.Local())
```

Comment:
Maybe we should set the error to what this function returns, if the error is nil? i.e. something like:

```
defer func() {
  qerr := Quarantine(ctx, ...)
  if *err == nil {
    err = qerr
  }
}(&err)
```